src/simularium/SelectionInterface.ts

```diff
@@ -1,8 +1,11 @@
-import { filter, find, uniq } from "lodash";
+import { filter, find, set, uniq } from "lodash";
 import { EncodedTypeMapping } from "./types";
 import { convertColorNumberToString } from "../visGeometry/color-utils";
+import _ from "lodash";
```

Comment:
can remove

Reply:
oops, fixed this and the two other things the linter caught